src/Orckestra.Composer.Cart/ViewModels/LineItemDetailViewModel.cs

```diff
@@ -48,13 +48,13 @@ public sealed class LineItemDetailViewModel : BaseViewModel
         /// The price for one instance of the item (quantity = 1), at the moment that the item is to be processed by the workflow
         /// </summary>
         [Formatting("General", "PriceFormat")]
-        public string ListPrice { get; set; }
+        public string CurrentPrice { get; set; }
```

Comment:
Please leave old fields with comment that they are deprecated to not break client solutions and just add  new ones